### DISCUSSION TITLE
Why does SvelteKit load 4 `about:blank` pages when adding Sentry?

### DISCUSSION THREAD

**Discussion by kevinrenskers** [sveltejs/kit#9262] [Q&A]
_I originally asked this [on the Sentry repo](https://github.com/getsentry/sentry-javascript/discussions/7289), but they say it's not them making these requests, it must be SvelteKit. Maybe somewhere here knows?_

I started a completely blank SvelteKit project, by using `npm create svelte@latest my-app` and choosing the skeleton option.

The only change I made is adding this code to `hooks.client.ts`:

```
import * as SentrySvelte from "@sentry/svelte";

SentrySvelte.init({
  dsn: "https://my_dsn_here",
});
```

And now I can see 4 requests getting made to `about:blank`, and I just can't figure out why this is happening. It seems to only happen in Safari, or at least it's the only browser that shows these in the network inspector.

![Screenshot 2023-02-27 at 20 36 41](https://user-images.githubusercontent.com/229384/221665692-e5f0166d-3409-4a4c-a4a5-4a8a2088fb05.png)

It's not a huge deal of course, but I just can't figure out what line of code is causing this. Creating a build of the project and grepping for `about:blank` doesn't give any results.

**Comment by filipwiniarski** [comment]
+1, curious about that one. I'm not sure if it affects performance, but I'm trying to optimize my app for Safari
